Monika After Story/game/script-greetings.rpy

```diff
@@ -3048,6 +3048,9 @@ label greeting_back_from_school:
 
     return
 
+default persistent._mas_pm_got_promotion = None
```

Comment:
I might be being picky, but to me this var reads as "Has the player been promoted?" instead of "When the last time the player's been promoted?". What do you think if we named it `persistent._mas_pm_last_promotion_d` or something?